doc/nrf/ug_radio_coex.rst

```diff
@@ -0,0 +1,196 @@
+
+This guide describes how to add WiFi Coexistence (CX) support to your SR application in |NCS|. In order to do it, both
+:ref:`Software support` and :ref:`Hardware description` must be added.
+
+Software support
+****************
+
+If your application uses SR protocols and requires coexistence with WiFi, you must enable :ref:`nrfxlib:mpsl_cx` in the
+:ref:`nrfxlib:mpsl` (MPSL) library.
+
+CX with MPSL
+============
+
+The MPSL library provides an implementation for the WiFi Coexistens module for SR applications.
+To use this implementation, your application must use a protocol driver that enables the WiFi Coexistence (CX) feature.
+The library provides multiprotocol support, but you can also use it in applications that require only one protocol.
+To avoid conflicts, check the protocol documentation to see if the protocol uses CX support provided by MPSL.
+
+.. note::
+   Currently, the following protocols use CX support provided by MPSL:
+
+   * :ref:`ug_thread`
+   * :ref:`ug_zigbee`
+
+.. _ug_radio_cx_requirements:
+
+Enabling MPSL
+-------------
+
+Before you add the devicetree node in your application, complete the following steps:
+
+   The MPSL library provides API to handle signals coming from PTA and controlling requests send to PTA.
+   See :ref:`nrfxlib:mpsl_lib` in the nrfxlib documentation for details.
+
+1. Add support for the MPSL library in your application.
+2. Enable support for MPSL implementation in |NCS| by setting the :kconfig:`CONFIG_MPSL` Kconfig option to ``y``.
+
+Enabling CX
+-------------
+
+In order to enable CX:
+
+1. Set the :kconfig:`CONFIG_MPSL_CX` Kconfig option to ``y``.
+2. Enable support for specific Coex implementation. The currently supported implementations are "generic 3-pin interface" and "thread", for
+   details see :ref:`WiFi Coex implementations`.
+#. On nRF53 devices, you must also apply the same Kconfig options mentioned in step 2 to the network core - see :ref:`_ug_multi_image`.
+
+Hardware description
+********************
+
+The |NCS| provides a wrapper that configures WiFi Coexistence based on devicetree (DTS) and Kconfig information.
+
+To enable CX support for any for the currently supported CX implementation, you must add an ``nrf_radio_coex`` node in the devicetree file, as
+described below. The node can also be provided by the devicetree file of the target development kit or by an overlay file. See
+:ref:`zephyr:dt-guide` for more information about the DTS data structure, and :ref:`zephyr:dt_vs_kconfig` for information about differences
+between DTS and Kconfig.
+
+For the currently supported CX implementation the ``nrf_radio_coex`` node has the ``compatible`` property set to
+``generic-radio-coex-three-wire``. Dts node compatible with ``generic-radio-coex-three-wire`` are suitable for a situation in which a 3 wire
+interface to the PTA is provided, ideally such as the one shown in the image below. However, the role of each of the pins is dependent on the Wifi
+Coexistence implementation.
```

Comment:
`is dependent on the Wifi Coexistence implementation.` ---> `depends on the implementation of the PTA.`

Reply:
ok